rocketpy/motors/hybrid_motor.py

```diff
@@ -345,18 +348,19 @@ def exhaust_velocity(self):
     @funcify_method("Time (s)", "Mass (kg)")
     def propellant_mass(self):
         """Evaluates the total propellant mass of the motor as the sum
-        of each tank mass and the grains mass.
```

Comment:
Just to give an idea....

"tank mass" is a perfect example of a bad documentation.
The tanks have no mass in rocketpy's model concept.
The person was trying to say "the mass of fluids contained in the tank". 

Text precision is crucial for a good documentation.